### PR TITLE
i18n(tr): update `getting-started.mdx` file

### PR DESCRIPTION
**Pull request by erkanaltinors**
**What kind of changes does this PR include?**

Updates of Starlight docs site content

**Description**

This pull request updates translated in Turkish getting-started.mdx file in project.

### CODE REVIEW

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few minor suggestions but overall this is looking really good! :thumbsup:

File: docs/src/content/docs/tr/getting-started.mdx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
-Aşağıdaki komutları kullanarak yeni bir Astro + Starlight projesi oluşturabilirsin:
+## Hızlı Başlangıç
+
+### Yeni Proje Oluşturma
+
+Aşağıdaki komutu terminalinde çalıştırarak yeni bir Astro + Starlight projesi oluştur:
 
 <Tabs>
 <TabItem label="npm">
```
[the remark this answers is not in the document]
Comment (erkanaltinors) [reply docs/src/content/docs/tr/getting-started.mdx:19]: Hello @HiDeoo , thanks for your kins response. I've removed comments for each installation commands with my [latest commit](https://github.com/withastro/starlight/pull/1792/commits/f828c317ddad77056119178dba205d99c0e60c55).

**Review by HiDeoo — COMMENTED** [review]
Thanks for the update, addresses perfectly all my comments 👍 

Unfortunately, this would need 1 more update as Starlight 0.22 was released yesterday and in the process, this page has been updated with [these changes](https://github.com/withastro/starlight/pull/640/files#diff-130a89ed8dd4068e1a366e0a6a31adb124cc60387ba42faadf457e83f9583948).
Fortunately, the changes are minimal and the idea is basically to add `syncKey="pkg"` to the 3 `<Tabs>` components in this page.

**Review by HiDeoo — APPROVED** [review]
This looks perfect, thanks a lot for the updates :raised_hands:

**Pull request merged**